templates/components/products/product-view.html

```diff
@@ -50,12 +50,14 @@ <h2 class="productView-brand"{{#if schema}} itemprop="brand" itemscope itemtype=
                     </span>
                 {{/if}}
                 {{#if settings.show_product_reviews}}
-                    <span class="productView-reviewLink">
+                    <button class="productView-reviewLink">
                         <a href="{{product.url}}{{#if is_ajax}}#write_review{{/if}}"
-                           {{#unless is_ajax }}data-reveal-id="modal-review-form"{{/unless}}>
+                           {{#unless is_ajax}}data-reveal-id="modal-review-form"{{/unless}}
+                           role="button"
+                        >
                            {{lang 'products.reviews.new'}}
                         </a>
-                    </span>
+                    </button>
```

Comment:
We have link nested in button here. As I know it can be issue for Firefox. Can you please check it? 
And here is the link that also can be helpful
https://stackoverflow.com/questions/16280684/nesting-a-inside-button-doesnt-work-in-firefox
Inside button our link also have role button. Maybe we can just use span for productView-reviewLink and change inner link to button?

Reply:
Tested in FF 78 and it works fine. In this case we need to use link because of href attribute usage and set role='button' for correct voiceover handling